modules/meshrenderinggl/include/modules/meshrenderinggl/datastructures/rasterization.h

```diff
@@ -31,20 +31,49 @@
 
 #include <modules/meshrenderinggl/meshrenderingglmoduledefine.h>
 #include <modules/opengl/shader/shaderutils.h>
+#include <modules/base/processors/transform.h>
 
 #include <vector>
 #include <unordered_map>
 #include <optional>
 
 namespace inviwo {
 
+namespace rasterization {
+
+struct SimpleSpatialEntity : public SpatialEntity<3> {

```

Comment:
Not sure what the purpose of this one is?

Reply:
I want to hand a transform around. Currently only in use in the rasterizations, which receive additinal transforms from either LineRasterizer, MeshRasterizer or the TransformRasterization processors. 
Generally, I'd only need a Model and World matrix, but using a light-weight SpatialEntity has the advantage of being able to set it directly. 
[See use here.](https://github.com/inviwo/inviwo/blob/d54984153cdc7b28f85fd3c4746b9228eb8f05f3/modules/meshrenderinggl/src/processors/meshrasterizer.cpp#L479)

I will add documentation there.

Reply:
I don't quite understand why you need both a world and model matrix though? when used in this way?

Reply:
Updated as we discussed at fika.